R/rownames.R

```diff
@@ -65,6 +65,23 @@ rownames_to_column <- function(df, var = "rowname") {
   new_df
 }
 
+#' @export
+#' @rdname rownames
+rowid_to_column <- function(df, var = "rowid") {
+  stopifnot(is.data.frame(df))
+  
+  if (has_name(df, var))
+    stopc("There is a column named ", var, " already!")
+  
+  if(nrow(df) == 0)
+    stopc('Data frame must have 1 or more rows')
+  
+  rn <- tibble(seq(1, nrow(df)))
```

Comment:
`seq_len()` also works for the corner case of zero rows, which I'd like to support.